backend/drtrottoir/tests/test_schedule_assignment.py

```diff
@@ -518,6 +519,22 @@ def test_schedule_assignment_delete_allowed_admin() -> None:
     assert response_admin.status_code == 204
 
 
+@pytest.mark.django_db
+def test_schedule_assignment_delete_not_allowed_work_entries() -> None:
```

Comment:
Could you rename this to `test_schedule_assignment_delete_not_allowed_with_existing_work_entries`? I think that'd be a clearer name for the test.